staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -400,15 +402,15 @@ type JobStatus struct {
 	Ready *int32 `json:"ready,omitempty" protobuf:"varint,9,opt,name=ready"`
 }
 
-// UncountedTerminatedPods holds UIDs of Pods that have terminated but haven't
```

Comment:
revert

Reply:
Is this because it is considered a type?